packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionGeneral.tsx

```diff
@@ -23,6 +31,88 @@ export const WfoSubscriptionGeneral = ({
     subscriptionDetail,
 }: WfoSubscriptionGeneralProps) => {
     const t = useTranslations('subscriptions.detail');
+    const [setSubscriptionInSync] = useSetSubscriptionInSyncMutation();
+    const { showToastMessage } = useShowToastMessage();
+    const { showConfirmDialog } = useContext(ConfirmationDialogContext);
+
+    const setInSyncAction = () => {
+        setSubscriptionInSync(subscriptionDetail.subscriptionId)
+            .unwrap()
+            .then(() => {
+                // Optimistic update for now
+                showToastMessage(
+                    ToastTypes.SUCCESS,
+                    t('setInSyncSuccess.text'),
+                    t('setInSyncSuccess.title'),
+                );
+                subscriptionDetail.insync = true;
+            })
+            .catch((error) => {
+                showToastMessage(
+                    ToastTypes.ERROR,
+                    error?.data?.detail
+                        ? error.data.detail
+                        : t('setInSyncFailed.text').toString,
+                    t('setInSyncFailed.title'),
+                );
+                console.error('Failed to set subscription in sync.', error);
+            });
+    };
+
+    const InSyncField = ({ inSync }: { inSync: boolean }) => {
+        const lastTaskRunDate = getLatestTaskDate(
+            subscriptionDetail.processes.page,
+        );
+        const lastUncompletedProcess = getLastUncompletedProcess(
+            subscriptionDetail.processes.page,
+        );
+
+        const getProcessLink = () => {
+            const processUrl =
+                (lastUncompletedProcess?.isTask ? PATH_TASKS : PATH_WORKFLOWS) +
+                '/' +
+                lastUncompletedProcess?.processId;
+
+            const confirmSetInSync = () => {
+                showConfirmDialog({
+                    question: t('setInSyncQuestion'),
+                    confirmAction: () => {
+                        setInSyncAction();
+                    },
+                });
+            };
+
+            return (
+                <>
+                    <Link
+                        href={processUrl}
+                        css={{ paddingLeft: 10, paddingRight: 20 }}
+                    >
+                        {t('see')} {lastUncompletedProcess?.processId}
+                    </Link>
+                    <EuiButton
+                        color="danger"
+                        size="s"
+                        onClick={confirmSetInSync}
+                    >
+                        {t('setInSync')}
+                    </EuiButton>
+                </>
+            );
+        };
+
+        return (
+            <>
+                <div css={{ paddingRight: 4, display: 'flex' }}>
```

Comment:
Try to use values specified in the theme

Reply:
Done